python/example_robot_data/robots_loader.py

```diff
@@ -258,21 +258,21 @@ def loadHyQ():
     return HyQLoader().robot
 
 
-class SoloLoader(RobotLoader):
+class Solo8Loader(RobotLoader):
```

Comment:
The main `ROBOTS` dict still contain `SoloLoader`, and I think it would be a good thing to add it for a deprecation message, eg.:

```python
class SoloLoader(Solo8Loader):
    def __init__(self, *args, **kwargs):
        warnings.warn('"solo" is deprecated, please try to load "solo8" or "solo12"')
        return super(SoloLoader, self).__init__(*args, **kwargs)
```


(and keep the change you made to the new class)